src/resource/resource-opening-hours/opening-period/OpeningPeriod.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { DatePeriod } from '../../../common/lib/types';
+import { formatDateRange } from '../../../common/utils/date-time/format';
+import './OpeningPeriod.scss';
+
+export default function OpeningPeriod({
+  datePeriod,
+}: {
+  datePeriod: DatePeriod;
+}): JSX.Element {
+  return (
+    <div className="opening-period-card">
```

Comment:
Should we leave the -card postfix, as this is no longer opening-period card? If I understood correctly, the new design does not use cards?

Reply:
I can remove it